### ISSUE TITLE
NotImplementedError when running 'py3-cmd refresh'

### ISSUE THREAD

**Issue by btmcg** [ultrabug/py3status#2003]
A recent upgrade to v3.32 started producing `NotImplementedError`s when I run `py3-cmd refresh`. Whether I choose `--all` or a specific module, the result is the same.

Happy to provide more info on request.

```
$ uname -a
Linux eolas 5.10.12-arch1-1 #1 SMP PREEMPT Sun, 31 Jan 2021 00:41:06 +0000 x86_64 GNU/Linux
$ py3-cmd --version
py3status 3.32 (python 3.9.1)
$ py3-cmd refresh --all
Traceback (most recent call last):
  File "/usr/bin/py3-cmd", line 13, in <module>
    sys.exit(send_command())
  File "/usr/lib/python3.9/site-packages/py3status/command.py", line 534, in send_command
    for uds in uds_list:
  File "/usr/lib/python3.9/pathlib.py", line 1164, in glob
    raise NotImplementedError("Non-relative patterns are unsupported")
NotImplementedError: Non-relative patterns are unsupported
```

**Comment by ultrabug** [comment]
hi @btmcg 

Just acknowledging that I saw your issue and will look at it ASAP

ping @eumiro we broke something

**Comment by eumiro** [comment]
Right, this was introduced when converting to `pathlib.Path`. Absolute paths have to be `glob`bed differently. Sorry, here's a fix PR in #2005. Could we test it?

**Comment by ultrabug** [comment]
Thanks @eumiro I fixed it as well

**Comment by ultrabug** [comment]
@btmcg @choopm @dpeukert version 3.33 hotfix released!

**Comment by eumiro** [comment]
Thank you @ultrabug and sorry for the bug!

**Comment by btmcg** [comment]
Can confirm that issue is fixed in version 3.33. Thanks all for the quick response and fix!
